.github/RELEASE.md

```diff
@@ -0,0 +1,265 @@
+<!-- These docs are updated and synced from https://github.com/conda/infra -->
+
+## Release Process
+
+[epic template]: issues/new?assignees=&labels=epic&template=epic.yml
```

Comment:
This link doesn't seem to work, it goes to a 404 page (but maybe once this is merged it'll go directly to the file?)

Reply:
these links aren't expected to work until the file is synced out, its supposed to be a relative link

Reply:
**but** after further reflection I believe the relative URL will still be wrong, when we view this file on GH it'll be at, e.g., https://github.com/conda/conda/blob/main/RELEASE.md so we want:

```suggestion
[epic template]: ../../issues/new?assignees=&labels=epic&template=epic.yml
```